pkg/operator/saptuneapplysolution_v1_test.go

```diff
@@ -72,48 +57,176 @@ func TestSaptuneApplySolutionSuccess(t *testing.T) {
 		"test-op",
 		operator.OperatorOptions[operator.SaptuneApplySolution]{
 			OperatorOptions: []operator.Option[operator.SaptuneApplySolution]{
-				operator.Option[operator.SaptuneApplySolution](operator.WithCustomSaptuneExecutor(mockCmdExecutor)),
+				operator.Option[operator.SaptuneApplySolution](operator.WithSaptuneClient(suite.mockSaptuneClient)),
 			},
 		},
 	)
 
 	report := saptuneSolutionApplyOperator.Run(ctx)
 
-	expectedDiff := map[string]any{
-		"before": saptuneSolutionAppliedNoSolutionOutput,
-		"after":  saptuneSolutionAppliedHanaSolutionOutput,
-	}
+	suite.Nil(report.Success)
+	suite.Equal(operator.PLAN, report.Error.ErrorPhase)
+	suite.EqualValues("saptune version not supported", report.Error.Message)
+}
+
+func (suite *SaptuneApplySolutionOperatorTestSuite) TestSaptuneApplySolutionPlanErrorBecauseFailingToDetermineInitiallyAppliedSolution() {
```

Comment:
nitpick
damn, never say such a long test case name. I don't think we need to write all literal words hehe
Things like `Because`, `ToDetermine` etc.
`TestSaptuneApplySolutionPlanErrorGettingSolution`

Reply:
ehehe verbosity gonna verbose 😝 1a8fde3ced667801e4c745a57b005d8369ff8d30